frontend/model/contracts/chatroom.js

```diff
@@ -396,15 +376,15 @@ sbp('chelonia/defineContract', {
         const isMentionedMe = data.type === MESSAGE_TYPES.TEXT &&
           (newMessage.text.includes(mentions.me) || newMessage.text.includes(mentions.all))
 
-        messageReceivePostEffect({
+        await messageReceivePostEffect({
           contractID,
           messageHash: newMessage.hash,
           height: newMessage.height,
           text: newMessage.text,
-          isDMOrMention: isMentionedMe || getters.chatRoomAttributes.type === CHATROOM_TYPES.DIRECT_MESSAGE,
+          isDMOrMention: isMentionedMe || state.settings.type === CHATROOM_TYPES.DIRECT_MESSAGE,
```

Comment:
The type is stored in `state.attributes` not `state.settings`. This issue is repeated in several places here.

While fixing this, can you please restore the use of the getters that was here since we can now import them from `frontend/model/contracts/shared/getters/chatroom.js`?